terraform/modules/leech/variables.tf

```diff
@@ -56,3 +67,25 @@ variable "leech_bucket_location" {
   default     = "US"
   nullable    = false
 }
+
+variable "github_app_id" {
+  description = "The GitHub App id of the application"
+  type        = string
+}
+
+variable "github_install_id" {
+  description = "The GitHub App installation id of the application"
+  type        = string
+}
+
+variable "github_private_key_secret" {
+  description = "The secret containing the private key for the GitHub app. Format: google_secret_manager_secret_version.name (eg. projects/{{project}}/secrets/{{secret_id}}/versions/{{version}})"
+  type        = string
```

Comment:
nit: would be nice to have a validation block here to prevent misconfiguration (or someone putting the actual secret value here)

Reply:
ack, changing this to be two parts